javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/AbstractTest.java

```diff
@@ -45,6 +45,15 @@ protected boolean isJava9() {
         return System.getProperty("java.version").startsWith("9.");
     }
 
+    protected boolean isJava10() {
+        return System.getProperty("java.version").startsWith("10.");
+    }
+
+    protected boolean isJavaVersionAbove9() {
```

Comment:
The name is misleading: above 9 should not include version 9 I think. Maybe rename it as "isJavaVersion9OrAbove"